pandas/_libs/tslibs/ccalendar.pxd

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# cython: profile=False
+
+from cython cimport Py_ssize_t
+
+from numpy cimport int64_t, int32_t
+
+
+cpdef monthrange(int64_t year, Py_ssize_t month)
```

Comment:
so monthrange is actually used in a few places. changing to use this one? (similar with other methods you defined), e.g. its defined in ``tslib.pyx`` now (as well as  ``normalize_date``)

Reply:
It is.  I figured I'd way to update the other usages until after you pointed it out, keep the diff smaller for the first round.

Reply:
no I'd rather see the far reaching changes (if any) now. Ideally in a PR you change a *small* number of things, but clean them up everywhere (e.g. replace usages of monthrange). sure its not always possible, but then its not 'half-done'